views/includes/url_input.ejs

```diff
@@ -1,15 +1,13 @@
 <div class="row center">
-    <div class="col s12">
-        <form class="row" action="/s" method="POST">
-            <div class="input-field col s12">
-                <input name="longUrl" id="longUrl" type="url">
-                <label for="longUrl">Enter Your URL Here</label>
-                <span class="helper-text" data-error="wrong" data-success="right">
-                    Enter URL and press enter
-                </span>
-                <br>
-                <button class="btn waves-effect waves-light" type="submit" name="action">Short This!</button>
-            </div>
-        </form>
-    </div>
-</div>
\ No newline at end of file
+  <div class="col s12">
+    <form class="row" action="/s" method="POST">
+      <div class="input-field col s12">
+        <input name="longUrl" id="longUrl" type="url" required />
```

Comment:
Added required attribute to make the field mandatory to have some input of url type.